roadmap.md

```diff
@@ -90,3 +90,11 @@ This scenario loads a web page and prints it as a PDF. In spec terms, this invol
 
 - [x] Some items from the previous milestones
 - [x] [Printing to PDF as Base64-encoded string](https://w3c.github.io/webdriver-bidi/#command-browsingContext-print)
+
+### Web extensions
+
+This scenario loads a web extension and accesses its background page. In spec terms, this involves:
+
+- [x] Some items from the previous milestones
+- [ ] Loading a web extension.
```

Comment:
Support for loading unsigned web extensions would be super helpful for extension developers (e.g. as part of continuous integration, especially since Chrome Web Store doesn't provide an API to download a signed .crx). Can we be explicit about whether this is for signed vs. unsigned extensions?